src/Flicking.ts

```diff
@@ -110,6 +112,10 @@ class Flicking extends Component {
     this.viewport = new Viewport(this, this.options, this.triggerEvent);
     this.listenInput();
     this.listenResize();
+
+    if (this.options.useStatistics) {
```

Comment:
How about option name to be `collectStatistics`?
Because we're collecting stats in here, not using them.